config/casestudies/adeo.yml

```diff
@@ -87,8 +87,21 @@ schemas:
   description: Avro 1.9
   storage: Git repository where source code is. During release they are published to Confluent Schema Registry.
   registry: Confluent Schema Registry.
-  versioning: Versioning is based on git tags. The schema version pushed to Confluent Schema Registry matches the git tag version of the product. Every schema has a `version` information that matches with product tag version.
-  validation: Based on validation using Confluent Schema Registry.
+  versioning: |
+    Versioning is based on git tags. The schema version pushed to Confluent Schema Registry 
+    matches the git tag version of the product. Every schema has a `version` information 
+    that matches with product tag version.
+
+    Example Avro schema with version information:
```

Comment:
@derberg 
```Example Avro schema with version information:
    ```json
    {
      "namespace": "com.adeo.casestudy.costingrequest",
      "type": "record",
      "name": "CostingRequestPayload",
      "version": "1.1.0",
      "fields": [ ... ]
    }
```
Does this whole code goes inside the `versioning:` key?

Reply:
yup, well done

Reply:
Fixed `validation` with this commit a3fb042656d8a302f99a20f7ac21d2022e89a491